src/commands/sentinel-info-cache.json

```diff
@@ -12,6 +12,47 @@
             "SENTINEL",
             "ONLY_SENTINEL"
         ],
+        "reply_schema": {
+            "type": "array",
+            "description": "Last cached INFO output from masters and all its replicas.",
+            "minItems": 0,
+            "maxItems": 4294967295,
+            "items": [
+                {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "description": "The master name."
+                        },
+                        {
+                            "type": "array",
+                            "description": "The unix timestamp at which the logged command was processed.",
+                            "minItems": 0,
+                            "maxItems": 4294967295,
+                            "items": [
```

Comment:
@oranagra i don't think so... mostly because this should have been a map and not an array
```
/* Reply format:
         *   1.) master name
         *   2.) 1.) info from master
         *       2.) info from replica
         *       ...
         *   3.) other master name
         *   ...
         */
         ```